### PR TITLE
Update describenet for new net archs

### PR DESCRIPTION
**Pull request by almaudoh**
Update describenet to properly handle attention body and attention policy nets.

### CODE REVIEW

**Review by borg323 — COMMENTED** [review]
Can you also add default activation and whether SELU is used in attention policy?

File: src/lc0ctl/describenet.cc [hunk 1]
```diff
@@ -129,28 +143,67 @@ void ShowNetworkWeightsInfo(const pblczero::Net& weights) {
...
-  COUT << Justify("Filters")
-       << w.input().weights().params().size() / 2 / 112 / 9;
-  COUT << Justify("Policy") << (w.has_policy1() ? "Convolution" : "Dense");
-  if (!w.has_policy1()) {
-    int policy_channels = w.policy().biases().params().size() / 2;
-    if (policy_channels == 0) {
-      policy_channels = w.policy().bn_means().params().size() / 2;
+
+  const auto attn_body = w.encoder_size() > 0;
+  if (attn_body) {
```
Comment (mooskagh) [inline src/lc0ctl/describenet.cc:148]: Optional nit: this function is becoming long, and it's easy to split it, maybe do that?

File: src/lc0ctl/describenet.cc [hunk 2]
```diff
@@ -129,28 +143,67 @@ void ShowNetworkWeightsInfo(const pblczero::Net& weights) {
...
+           << w.pol_encoder(0).mha().q_b().params().size() / 2;
+      COUT << Justify("Policy encoder DFF")
+           << w.pol_encoder(0).ffn().dense1_b().params().size() / 2;
+    }
+    COUT << Justify("Policy DModel") << w.ip2_pol_b().params().size() / 2;
+    if (attn_body) {
+      COUT << Justify("Policy activation")
+           << NetworkFormat::ActivationFunction_Name(
+                  NetworkFormat::ACTIVATION_DEFAULT);
+      COUT << Justify("Policy FFN activation")
```
Comment (borg323) [inline src/lc0ctl/describenet.cc:184]: The "Policy FFN activation" should only be printed if there are policy encoders. Probably cleaner to move "Policy activation" before the `if (format.policy() == NetworkFormat::POLICY_ATTENTION)`.

File: src/lc0ctl/describenet.cc (hunk 1 above)
Comment (almaudoh) [reply src/lc0ctl/describenet.cc]: Done

**Review by borg323 — APPROVED** [review]

**Pull request merged**